api/registrations/views.py

```diff
@@ -149,7 +149,7 @@ class RegistrationDetail(JSONAPIBaseView, generics.RetrieveUpdateAPIView, Regist
     """
     permission_classes = (
         drf_permissions.IsAuthenticatedOrReadOnly,
-        AdminOrPublic,
+        ContributorOrPublic,
```

Comment:
Dear reviewer, I struggled with this change decision. It seems as though there aren't really a lot of other things that can be done to a registration, but if we wanted to let write contributors add tags then this was indeed the spirit of the change. Are there other places that an admin is only allowed to do things other than changing from public to private?

Reply:
Drive-by: Custom Citations? They're currently admin only I believe.